planning/fil_record_interpreter.go

```diff
@@ -0,0 +1,89 @@
+package planning
+
+import (
+	"errors"
+	"github.com/filecoin-project/indexer-reference-provider/metadata"
+	"github.com/filecoin-project/storetheindex/api/v0"
+	"github.com/ipfs-shipyard/w3rc/contentrouting"
+)
+
+// ------- FORK START --------------
+// TODO: Remove forked code below once #14 is merged.
+// Forked to unblock implementation.
+// See: https://github.com/ipfs-shipyard/w3rc/pull/14
+type (
```

Comment:
Pending #14 merge to `main`. Forking code for now to unblock work and reduce merge conflicts